molo/usermetadata/views.py

```diff
@@ -0,0 +1,31 @@
+from django.views.generic import TemplateView
+from django.shortcuts import get_object_or_404
+from django.http import HttpResponseRedirect
+
+from molo.usermetadata.models import PersonaPage
+
+
+class PersonaView(TemplateView):
+    template_name = 'persona.html'
+
+    def get_context_data(self, *args, **kwargs):
+        persona_pages = PersonaPage.objects.live().all()
+        context = super(PersonaView, self).get_context_data(*args, **kwargs)
+        locale_code = context.get('locale_code')
+        context.update({
+            'persona_pages':
+            [a.get_translation_for(locale_code) or a for a in persona_pages],
```

Comment:
When we add tests, can we add a test to check that we use translated pages if a language with `locale_code` exists, and use the original page if no language with `locale_code` exists?